[2022-12-04T16:14:47.544+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.generate_data scheduled__2022-12-04T14:00:00+00:00 [queued]>
[2022-12-04T16:14:47.551+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.generate_data scheduled__2022-12-04T14:00:00+00:00 [queued]>
[2022-12-04T16:14:47.553+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:14:47.554+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-04T16:14:47.556+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:14:47.565+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): generate_data> on 2022-12-04 14:00:00+00:00
[2022-12-04T16:14:47.576+0000] {standard_task_runner.py:55} INFO - Started process 621 to run task
[2022-12-04T16:14:47.583+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_generation', 'generate_data', 'scheduled__2022-12-04T14:00:00+00:00', '--job-id', '79', '--raw', '--subdir', 'DAGS_FOLDER/generate_data.py', '--cfg-path', '/tmp/tmpl5a1p4vw']
[2022-12-04T16:14:47.588+0000] {standard_task_runner.py:83} INFO - Job 79: Subtask generate_data
[2022-12-04T16:14:47.681+0000] {task_command.py:389} INFO - Running <TaskInstance: data_generation.generate_data scheduled__2022-12-04T14:00:00+00:00 [running]> on host c7dedbf79ea9
[2022-12-04T16:14:47.744+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_generation
AIRFLOW_CTX_TASK_ID=generate_data
AIRFLOW_CTX_EXECUTION_DATE=2022-12-04T14:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-04T14:00:00+00:00
[2022-12-04T16:14:47.989+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-12-04T16:14:48.000+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=data_generation, task_id=generate_data, execution_date=20221204T140000, start_date=20221204T161447, end_date=20221204T161448
[2022-12-04T16:14:48.041+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-04T16:14:48.062+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
